server/command.go

```
@@ -189,9 +189,6 @@ func executeConnect(p *Plugin, context *model.CommandArgs, _ ...string) *model.C
 	if err == nil && len(conn.ConfluenceAccountID()) != 0 {
 		return p.responsef(context,
 			"You already have a Confluence account linked to your Mattermost account. Please use `/confluence disconnect` to disconnect.")
-	} else if err != nil {
```

Returns error only in case of no connection exists. So this error is ignored purposely.